_sass/_typography.scss

```diff
@@ -0,0 +1,64 @@
+/*  typography */
+@import url('https://fonts.googleapis.com/css?family=Raleway+Display:400,700|Roboto:300,400,500,700');
```

Comment:
Interesting, this theme uses a font import from Google, but just under this, it also loads a font from our static directory. :thinking: